dev/conductor/core/lib/src/start.dart

```diff
@@ -257,10 +256,18 @@ class StartContext {
   /// If validations should be overridden.
   final bool force;
 
+  late final EngineRepository engine;
```

Comment:
can you make these final and instantiate them in the constructor's initializer list? if not, I think it would be better to make these nullable and defer any null-pointer exceptions to the caller, rather than allowing it to happen in the core (any time you use `late final` without an initializer, it means you're actually breaking null-safety and the compiler cannot guarantee whether or not the value will be non-null).